src/types/PageChangeCallback.ts

```diff
@@ -0,0 +1 @@
+export type PageGhangeCallback = (currentState: number) => number;
```

Comment:
There seems to be a typo in the type name `PageGhangeCallback`. It should likely be `PageChangeCallback` to correctly represent its functionality and follow the naming convention.